cmd/fyne/internal/commands/package-windows.go

```diff
@@ -131,8 +135,15 @@ func (p *Packager) packageWindows(tags []string) error {
 func runAsAdminWindows(args ...string) error {
 	cmd := "\"/c\""
 
-	for _, arg := range args {
-		cmd += ",\"" + arg + "\""
+	for idx, arg := range args {
+		if idx == 0 {
```

Comment:
This can't be right? Only adding a comma for 0 index?
I think this would benefit from a unit test showing the correct format.

Reply:
Actually cmd.exe does parse the string given to it as the command to execute. So it means the entire arguments list has to be in the same argument passed to it. It is closed at the end of the loop, if there was more than one. I will see to add a unit test.

Reply:
As requested above can you please add a unit test?
The mix of single and double quotes is hard to parse here. With args "1" and "2" it looks like it will produce: 
* (before) `"/c","1","2"`
* (after) `"/c" ,'1 "2"'`

Without seeing the expected output I can't figure if this is desired...